circuit-lib/circuitlib-rs/circuits/merkle/MerkleTreeProof.circom

```diff
@@ -0,0 +1,49 @@
+pragma circom 2.1.4;
+
+include "../../node_modules/circomlib/circuits/poseidon.circom";
+include "../../node_modules/circomlib/circuits/comparators.circom";
+include "../../node_modules/circomlib/circuits/bitify.circom";
+include "../../node_modules/circomlib/circuits/switcher.circom";
+
+template MerkleTreeProof(levels, numberOfUTXOs) {
```

Comment:
we have this template in circuit lib right?

Reply:
in the Merkle tree folder I believe

Reply:
Yes, but it doesn't support variable number of utxos. Should we replace it new one?